### PR TITLE
Update "Get your 3D Model" page for Vircadia

### PR DESCRIPTION
**Pull request by jug007**
Removed mentions of the Marketplace and High Fidelity
Not all VR headsets run at 90Hz
Removed mention of uploading, as it suggest a central server you upload your content to.
Removed mention of proof of purchase certificates (I would guess that those are not a thing anymore)

### CODE REVIEW

**Comment by jug007** [comment]
#19 added gLTF here, but didn't check other docs pages for similar need.

File: docs/source/create/3d-models/get-model.rst [hunk 1]
```diff
@@ -13,17 +13,17 @@ Get Your 3D Model from 3D Content Stores
...
-+ **Check Materials:** You might find that the model may be missing its textures. If that happens, first check to see if the textures are included. If a model loads into High Fidelity and doesn't look right, you may also find error information in the Interface logs.  
++ **Check Licensing Terms:** Make sure you check a model's licensing terms before you use it. It is your responsibility to ensure that you have sufficient rights to use the content. When you make a 3D model available on your Vircadia server, visitors are getting the links to those files in the same way as they would when viewing an image on a website. You should be comfortable and have the rights to re-distribute the 3D content.
```
Comment (kasenvr) [inline docs/source/create/3d-models/get-model.rst:16]:
```suggestion
+ **Check Licensing Terms:** Make sure you check a model's licensing terms before you use it. It is your responsibility to ensure that you have sufficient rights to use the content. When you make a 3D model available on your Vircadia server, visitors are getting the links to those files in the same way as they would when viewing an image on a website. You should be comfortable that you have the rights to re-distribute the 3D content.
```

File: docs/source/create/3d-models/get-model.rst [hunk 2]
```diff
@@ -2,7 +2,7 @@
 Get Your 3D Model
 #########################
 
-Many 3D models are available for purchase on our `Marketplace <../../explore/shop.html#shopping-the-marketplace>`_. If the Marketplace doesn't have the model you're looking for, you can create and customize your own. All 3D models should be in the OBJ or FBX format and have materials supported by High Fidelity.  
+All 3D models should be in the FBX, gLTF or OBJ format and have materials supported by Vircadia.
```
Comment (ctrlaltdavid) [inline docs/source/create/3d-models/get-model.rst:5]:
```suggestion
All 3D models should be in the FBX, glTF or OBJ format and have materials supported by Vircadia.  
```

File: docs/source/create/3d-models/get-model.rst [hunk 3]
```diff
@@ -13,17 +13,17 @@ Get Your 3D Model from 3D Content Stores
...
 
 ---------------------------------
 Create Your Own 3D Model
 ---------------------------------
 
 You can create your own 3D model using 3D modeling software such as Blender or Maya. Use any software of your choice as long as:
 
-+ The 3D model is in the OBJ or FBX format.
-+ The 3D model materials are supported by High Fidelity. Use our :doc:`materials guide <pbr-materials-guide>` to make sure that your materials load correctly.
++ The 3D model is in the FBX, gLTF or OBJ format.
```
Comment (ctrlaltdavid) [inline docs/source/create/3d-models/get-model.rst:25]:
```suggestion
+ The 3D model is in the FBX, glTF or OBJ format.
```

**Comment by jug007** [comment]
I will try to keep the glTF spelling in mind for future docs.

**Pull request merged**